Source/driver/Castro_advance.cpp

```diff
@@ -501,6 +501,16 @@ Castro::finalize_advance()
         FluxRegFineAdd();
     }
 
+    // The mass_fluxes array currently holds only the fluxes
+    // from the last subcycle. Override this with the sum of
+    // the fluxes from the full timestep (this will be used
+    // later during the reflux operation).
+
```

Comment:
the PR says this change makes mass_fluxes hold the full timestep's worth of fluxes -- doesn't this comment say the opposite?

Reply:
It is assuming that `fluxes`, which is about to be copied to `mass_fluxes`, has the full timestep's worth of flux data at this point.

Reply:
ah, I understand.